xlgui/main.py

```diff
@@ -1074,7 +1074,7 @@ def window_state_change_event(self, window, event):
             # else
             #    destroy tray
 
-            if self.minimized != prev_minimized and self.minimized == True:
+            if self.minimized != prev_minimized and self.minimized:
```

Comment:
The reason I prefer the explicit comparison here is because the expression `self.minimized` is already in the first part of the condition (before the `and`), so the instance in the second part is like something is lost after that. Anyway, it's only a matter of personal taste/style.